### PR TITLE
removed class_roster block from base.py and added it to aws.py with b…

### PR DESCRIPTION
**Pull request by eparker71**
…rackets [] and local.py with default values.

@bermudezjd @sapnamysore @elliottyates

### CODE REVIEW

File: canvas_course_admin_tools/settings/local.py [hunk 1]
```diff
@@ -26,6 +26,15 @@
     'https://icommons-tools.dev.tlt.harvard.edu/course_conclusion'
 )
 
+CLASS_ROSTER = {
+    'sis_roster': {
+        'base_path': SECURE_SETTINGS.get('sis_roster_base_path', 'psp/hrvcsint'),
+        'static_path': '/EMPLOYEE/HRMS/c/SA_LEARNING_MANAGEMENT.SS_CLASS_ROSTER.GBL',
+        'base_url': SECURE_SETTINGS.get('sis_roster_base_url', 'https://csint.my.harvard.edu/'),
```
Comment (bermudezjd) [inline canvas_course_admin_tools/settings/local.py:33]: This and the `base_path` settings are fine if you want the ability to go back-and-forth between hitting the production and staging/test endpoints of the tool, but if you are only ever going to go against the stage endpoint then you could alternatively hardcode those settings.

**Comment by bermudezjd** [comment]
This should be fine for releasing tomorrow 👍   I would suggest, however, that you remove the `sis_roster` child dictionary from the `CLASS_ROSTER` dictionary and instead insert keys directly into the latter with an `sis_roster` prefix if desired.  That way, references would be something like `settings.CLASS_ROSTER['sis_roster_base_path']` instead of `settings.CLASS_ROSTER['sis_roster']['base_path']`.

**Comment by elliottyates** [comment]
Thanks @bermudezjd. I will merge for now. Why the recommendation to split off settings into individual bits like that, for the convenience of overriding them? Or just because we have no other subdicts in `CLASS_ROSTER` at this time?

**Comment by bermudezjd** [comment]
@elliottyates It just seems unnecessary to have any subdicts since you can qualify bits of the class roster app by using more descriptive keys if needed.  Having a single layer would make it easier to override, and it should be a little more readable and concise where the settings are being accessed.

**Pull request merged**